apps/files/lib/Command/Scan.php

```diff
@@ -289,10 +289,9 @@ protected function showSummary($headers, $rows, OutputInterface $output) {
 	 * @return string
 	 */
 	protected function formatExecTime() {
-		list($secs, ) = explode('.', sprintf("%.1f", $this->execTime));
-
-		# if you want to have microseconds add this:   . '.' . $tens;
-		return date('H:i:s', $secs);
+		$secs = round($this->execTime);
+		# convert seconds into HH:MM:SS form
+		return sprintf('%02d:%02d:%02d', ($secs/3600),($secs/60%60), $secs%60);
```

Comment:
```suggestion
		return sprintf('%02d:%02d:%02d', ($secs/3600), ($secs/60%60), $secs%60);
```

There is one space missing :laughing:

Reply:
> Looks good to me. There is a similar logic in:
> 
> [server/apps/files/lib/Command/ScanAppData.php](https://github.com/nextcloud/server/blob/1ad42e8f81dd6f94c205af9cd3b5cb347f803dd4/apps/files/lib/Command/ScanAppData.php#L219)
> 
> Line 219 in [1ad42e8](/nextcloud/server/commit/1ad42e8f81dd6f94c205af9cd3b5cb347f803dd4)
>  		list($secs, ) = explode('.', sprintf("%.1f", $this->execTime)); 
> Would be nice to have them in sync.

Ok :+1: Let me change it too!